judge/models/contest.py

```diff
@@ -134,11 +134,19 @@ class Contest(models.Model):
     hide_problem_tags = models.BooleanField(verbose_name=_('hide problem tags'),
                                             help_text=_('Whether problem tags should be hidden by default.'),
                                             default=False)
+    hide_problem_authors = models.BooleanField(verbose_name=_('hide problem authors'),
+                                               help_text=_('Whether problem authors should be hidden by default.'),
+                                               default=False)
     run_pretests_only = models.BooleanField(verbose_name=_('run pretests only'),
                                             help_text=_('Whether judges should grade pretests only, versus all '
                                                         'testcases. Commonly set during a contest, then unset '
                                                         'prior to rejudging user submissions when the contest ends.'),
                                             default=False)
+    show_short_display = models.BooleanField(verbose_name=_('show short form settings display'),
+                                             help_text=_('Whether to show a section containing contest settings '
+                                                         'on the contest page or not.'),
+                                             default=False)
+    is_organization_private = models.BooleanField(verbose_name=_('private to organizations'), default=False)
```

Comment:
We already have this above. It's uses elsewhere should be looked at carefully to (though I think most are unit tested which is nice).

Reply:
The duplicate is removed. The other uses seem fine.